app/routes.py

```diff
@@ -1 +1,291 @@
+    API_KEY = "xoxb-4342317584466-4335756490678-gKLN3O9dpjHPgZ2aTJIxn5GX"
+    query_params = {
+        "channel": "task-notifications",
+        "text": message
+    }
+    query_header = {"Authorization":f"Bearer {API_KEY}"}
+    return requests.post(path,params = query_params, headers = query_header)
+
+
+####TASK#####
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    try:
+        request_body = request.get_json()
+        new_task = Task(title=request_body["title"],
+                    description=request_body["description"])
+    except KeyError as err:
+        missing_field_response = {"details": "Invalid data"}
+        return make_response(jsonify(missing_field_response), 400)
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    response = {"task":{
+                "description": new_task.description,
+                "id": new_task.task_id,
+                "is_complete": False,
+                "title": new_task.title 
+            }}
+    return make_response(jsonify(response), 201)
+######GOAL#####
+@goals_bp.route("", methods=["POST"])
+def create_goal():
+    try:
+        request_body = request.get_json()
+        new_goal = Goal(title=request_body["title"])
+    except KeyError as err:
+        missing_field_response = {"details": "Invalid data"}
+        return make_response(jsonify(missing_field_response), 400)
+
+    db.session.add(new_goal)
+    db.session.commit()
+
+    response = {"goal":{
+                "id": new_goal.goal_id,
+                "title": new_goal.title 
+            }}
+    return make_response(jsonify(response), 201)   
+######TASK#####
+@tasks_bp.route("", methods=["GET"])
+def read_all_tasks():
+    order_by = request.args.get("sort")
+    if order_by == "asc":
+        tasks = Task.query.order_by(Task.title.asc()).all()
+    elif order_by == "desc":
+        tasks = Task.query.order_by(Task.title.desc()).all()
+    else:
+        tasks = Task.query.all()
+
+    tasks_response = []
+    for task in tasks:
+        tasks_response.append(
+            {
+                "id": task.task_id,
+                "title": task.title,
+                "description": task.description,
+                "is_complete": False,
+            }
+        )
+    return jsonify(tasks_response)
+####GOAL####
+@goals_bp.route("", methods=["GET"])
+def read_all_goals():
+    goals = Goal.query.all()
+    goals_response = []
+    for goal in goals:
+        goals_response.append(
+            {
+                "id": goal.goal_id,
+                "title": goal.title,
+            }
+        )
+    return jsonify(goals_response)
+#####TASK####   
+@tasks_bp.route("/<task_id>", methods=["GET"])
+def read_one_task(task_id):
+    task = validate_task(task_id)
+    if not task.goal_id:
+        return {"task":{
+                    "id": task.task_id,
+                    "title": task.title,
+                    "description": task.description,
+                    "is_complete": False
```

Comment:
This suggests that tasks will never have a completed timestamp, which is not necessarily true. How can we check whether or not a task has a timestamp and thus should be `True` or `False`?